pkg/kubelet/pod_workers.go

```diff
@@ -626,6 +632,16 @@ func (p *podWorkers) UpdatePod(options UpdatePodOptions) {
 		// the channel is empty, so buffer of size 1 is enough.
 		podUpdates = make(chan podWork, 1)
 		p.podUpdates[uid] = podUpdates
+		fullname := kubecontainer.GetPodFullName(pod)
+		startedUID, started := p.startedPodUIDsByFullname[fullname]
+		if !started || startedUID == uid {
+			// we can start a pod
+			p.startedPodUIDsByFullname[fullname] = uid
```

Comment:
If this becomes only static pods, it should describe that it only applies to static pods.